### ISSUE TITLE
NTR: Cat scratch or bite possible several days before symptoms (Immune system assessed clinically)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8044]
New term request Cat scratch or bite possible several days before symptoms (samant_101230220058):   UMLS: 2170591

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000539
- label: History of recent cat bite
- parents: [History of recent animal bite (HP:0020138)](https://hpo.jax.org/app/browse/term/HP:0020138)
- pmid: 
- def: Medical history of a recent bite or scratch injury due to a cat.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8044
- synonyms: History of recent cat scratch(NONE)
